hedera-node/hedera-schedule-service-impl/build.gradle.kts

```diff
@@ -26,12 +26,12 @@ dependencies {
         testImplementation(testFixtures(project(":app-spi")))
         testImplementation(testFixtures(project(":config")))
         testImplementation(project(":app"))
+        testImplementation(project(":app-service-schedule-impl"))
```

Comment:
What drove this change?  It seems odd that a module would have to declare itself as a dependency...

Reply:
Yes. There was a similar remark from Hendrik (https://github.com/hashgraph/hedera-services/pull/7875#discussion_r1293804883). Thanks for catching this. The analysis gave some questionable results here and I followed that. Removed this.